sct.c

```diff
@@ -60,42 +59,88 @@ static const struct { float r; float g; float b; } whitepoints[] = {
     { 0.78988728,  0.86491137,  1.00000000, }, /* 10000K */
     { 0.77442176,  0.85453121,  1.00000000, },
 };
-
-static void
-sct_for_screen(Display *dpy, int screen, int temp)
+#define TEMPERATUREDEFAULT 6500
```

Comment:
```suggestion
#define TEMPERATURE_DEFAULT 6500
```
same for the vars below --- helps readability